New-york-times/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@24,400,0,0&icon_names=more_horiz"
+    />
+    <link
+      href="https://fonts.cdnfonts.com/css/english-towne"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="nav-container">
+        <nav class="nav-superior nav-grid">
+          <a class="icon-search" href="#">
+            <i class="fa-solid fa-magnifying-glass"></i>
+          </a>
+          <a class="icon-notification" href="#">
+            <i class="fa-regular fa-bell"></i>
+          </a>
+          <img
+            src="https://upload.wikimedia.org/wikipedia/commons/5/58/NewYorkTimes.svg"
+            alt="The New York Times Logo"
+            class="nyt-logo"
+          />
+          <a href="#" class="button-signin">SIGN IN</a>
+          <a href="#" class="button-subscribe">SUBSCRIBE</a>
+        </nav>
```

Comment:
Estos botones de SIGN IN y SUBSCRIBE están muy pegados al siguente bloque de abajo. Hay que aumentar el padding.

> [!CAUTION]
> Cuando se desarrollen pantallas en aplicaciones inube, se espera que el frontend sea exactamente igual a los diseños. Sí, aquí no hay figma como para poder tomar medidas, pero sí se puede hacer una aproximación mayor a lo que la imagen de referencia está mostrando.
>
> No se evalúa exclusivamente que se conozcan las etiquetas HTML las reglas para estilos CSS. También se evalúa la destreza en el uso de esos conocimientos para crear pantallas fieles a lo que el diseñador plantea.